lib/model/unittest/CModelTestFixtureBase.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+#ifndef INCLUDED_CModelTestFixtureBase_h
+#define INCLUDED_CModelTestFixtureBase_h
+
+#include <core/CoreTypes.h>
+
+#include <maths/CBasicStatistics.h>
+#include <maths/CModel.h>
+#include <maths/CMultivariatePrior.h>
+
+#include <model/CResourceMonitor.h>
```

Comment:
We probably ought not to rely on transitive includes of `std` and `boost` functionality here, i.e. `#include <utility>`, `#include <vector>`, etc